_posts/2020-03-07-Team-Retreat.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: Team Retreat: How They Can It Help Remote Teams "Click"
+date: 2020-03-12 14:30:00
+categories: [agile]
+author: abizzinotto
+
+---
+
+Remote work is a growing reality. In the 21st century, we have distributed companies that work 100% remotely, with team members scattered around the world.
+
+While remote work has its benefits - the diversity in background and experiences, access to an immense talent pool, cut down office costs to name a few - it also brings its challenges. And the biggest one is communication.
+
+There are several pieces of software that aim to faciliate communication in a remote work environment. At OmbuLabs, we use [Slack](https://slack.com) for our day-to-day conversation and [Zoom](https://zoom.us/) for our calls. However, as a remote team, we don't get to meet our teammates face to face. We don't have "water cooler" chats or as many opportunities for casual chats as an office environment.
+
+So how can we bring our team close together? How can we build that team spirit that comes from seeing every one and talking about our casual, social lives? Well, this month we had a team retreat. And it was great.
```

Comment:
@ABizzinotto Would be interesting if we can add a photo from our team retreat here?

Reply:
Good point. @lubc could you share the official photo we took on Saturday?

Reply:
@ABizzinotto I think he shared a few in Slack.

Reply:
Added.